src/modules/navigator/mission.h

```diff
@@ -240,6 +240,80 @@ class Mission : public MissionBlock, public ModuleParams
 
 	void publish_navigator_mission_item();
 
+
+	/**
+	* @brief Get the index associated with the last item that contains a position
+	* @param mission The mission to search
+	* @param start_index The index to start searching from
+	* @param prev_pos_index The index of the previous position item containing a position
+	* @return true if a previous position item was found
+	*/
+	bool getPreviousPositionItemIndex(const mission_s &mission, int start_index, unsigned &prev_pos_index);
```

Comment:
@sfuhrer Can these functions be made const?

Reply:
certainly, done